packages/nodes-base/nodes/Postgres/v2/actions/database/executeQuery.operation.ts

```diff
@@ -95,7 +85,11 @@ export async function execute(
 					const resolvables = getResolvables(rawValues);
 					if (resolvables.length) {
 						for (const resolvable of resolvables) {
-							const evaluatedValues = stringToArray(this.evaluateExpression(`${resolvable}`, i));
+							const evaluatedExpression = this.evaluateExpression(`${resolvable}`, index) as string;
```

Comment:
use toString instead of casting
```suggestion
							const evaluatedExpression = this.evaluateExpression(`${resolvable}`, index)?.toString()??'';
```